e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
In addition, I think it is better to name the file to something like `runE2EAgainstSauceLab.sh` cause it is specific to source lab?